UI/configs.py

```diff
@@ -0,0 +1,6 @@
+link = "https://iced-latte.uk/"
+first_name = "Test"
+last_name = "Testtest"
+email = "ilsinyakovmo4@gmail.com"
```

Comment:
@Sunagatov, Zufar, I need you advice on this

I believe that publicly committing real credentials that are used for authorization on production server is dangerous, I suggest replacing real email and password with empty strings. Yes, this will make tests fail until those strings are not replaced with real data locally. Once we prepare local sandboxes, we can commit any credentials

Reply:
I suggest to leave it as it is until our devops team provides a test environent in our server to us.